templates/base.html

```diff
@@ -40,15 +40,19 @@
   <!-- Open Graph -->
   {% set image = config.image[0].src if config.image else 'oddsite.jpg' %}
   {% set image = image if '://' in image else [config.canonical_url, 'static/images/blog/', image]|join('') %}
-  <meta property="og:title" content="{{ self.title() }} | OddBird" />
-  <meta property="og:type" content="website" />
+  <meta property="fb:app_id" content="1820980378150914" />
```

Comment:
I'm not sure this actually does anything, since our app is in "Development" mode and not public on FB. But we don't really want to make it public unless we actually want people to be linked there... Maybe this still gets us analytics? Not sure.